test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -1,7 +1,6 @@
 package seedu.addressbook.data;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
Imported classes should always be listed explicitly

Reply:
will fix